miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSequencerReferenceController.java

```diff
@@ -93,7 +93,7 @@ public ModelAndView viewSequencer(@PathVariable(value = "referenceId") Long refe
 
     model.put("sequencerReference", sr);
     model.put("sequencerServiceRecords", serviceRecords);
-    model.put("title", "Edit Sequencer");
+    model.put("title", "Sequencer " + sr.getId());
     String ip = sr.getIpAddress() == null ? "" : sr.getIpAddress().toString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "getIpAddress" returns a string, there's no need to call "toString()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)